src/persist-client/src/fetch.rs

```diff
@@ -1101,7 +1097,6 @@ where
 /// clone is very cheap.
 #[derive(Debug, Clone, Default)]
 pub(crate) struct Cursor {
-    part_idx: usize,
     idx: usize,
 }
```

Comment:
IMO it wouldn't be silly to replace `Cursor` with just a `usize` now wherever it's used... but that's a wider change that I'd prefer to kick to a future PR.